ArduCopter/ekf_check.cpp

```diff
@@ -121,11 +129,8 @@ bool Copter::ekf_over_threshold()
     } else if (vel_variance >= g.fs_ekf_thresh) {
         over_thresh_count++;
     }
-    if (position_variance >= g.fs_ekf_thresh) {
-        over_thresh_count++;
-    }
 
-    if (over_thresh_count >= 2) {
+    if (position_variance >= g.fs_ekf_thresh && over_thresh_count >= 1) {
```

Comment:
this change seems to preclude the possibility of an EKF failsafe due to large height variance

Reply:
See response below